mars-app-spa/src/FCs/CameraDropDownFC.tsx

```diff
@@ -0,0 +1,28 @@
+import { useContext, useEffect, useState } from "react";
+import { CameraI } from "../nasaInterfaces";
+import { selectedRoverContext } from "../selectedRoverContext";
+import { DropDownFC } from "./dropDownFC";
+
+export const CameraDropDownFC : React.FC = 
+(()=>{
+    const programContext = useContext(selectedRoverContext);
+    const [cameras, setCameras] = useState<CameraI[]>([]);

```

Comment:
Could initialize this to some dummy option - maybe with a `label` of `loading options`, in case the request takes too long

Reply:
we'll hide the camera drop down component if no rover is selected, so I think leaving the default value as an empty array is fine